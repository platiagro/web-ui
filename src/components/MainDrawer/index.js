import React from 'react';

import { Drawer } from 'antd';

const MainDrawer = ({ children, title }) => (
  <Drawer
    width={350}
    title={title}
    placement='right'
    closable
    onClose={null}
    visible
  >
    {children}
  </Drawer>
);

export default MainDrawer;
