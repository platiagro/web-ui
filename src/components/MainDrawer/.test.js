import React from 'react';

import { shallow } from 'enzyme';

import MainDrawer from '.';

describe('MainDrawer component', () => {
  it('renders without crashing', () => {
    shallow(<MainDrawer />);
  });

  it('renders html correctly', () => {
    const mainDrawerShalowed = shallow(<MainDrawer />);

    expect(mainDrawerShalowed).toMatchSnapshot();
  });
});
