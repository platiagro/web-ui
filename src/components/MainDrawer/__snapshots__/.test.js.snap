// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainDrawer component renders html correctly 1`] = `
<withConfigConsumer(Drawer)
  closable={true}
  onClose={null}
  placement="right"
  visible={true}
  width={350}
/>
`;
